src/statmodels.jl

```diff
@@ -1,220 +1,3 @@
-"""
-mss(model::StatisticalModel) = error("mss is not defined for $(typeof(model)).")
-
-"""
-    rss(model::StatisticalModel)
-
-Return the residual sum of squares of the model.
-"""
-rss(model::StatisticalModel) = error("rss is not defined for $(typeof(model)).")
-
-"""
-    informationmatrix(model::StatisticalModel; expected::Bool = true)
-
-Return the information matrix of the model. By default the Fisher information matrix
-is returned, while the observed information matrix can be requested with `expected = false`.
-"""
-informationmatrix(model::StatisticalModel; expected::Bool = true) =
-    error("informationmatrix is not defined for $(typeof(model)).")
-
-"""
-    stderror(model::StatisticalModel)
-
-Return the standard errors for the coefficients of the model.
-"""
-stderror(model::StatisticalModel) = sqrt.(diag(vcov(model)))
-
-"""
-    vcov(model::StatisticalModel)
-
-Return the variance-covariance matrix for the coefficients of the model.
-"""
-vcov(model::StatisticalModel) = error("vcov is not defined for $(typeof(model)).")
-
-"""
-    weights(model::StatisticalModel)
-
-Return the weights used in the model.
-"""
-weights(model::StatisticalModel) = error("weights is not defined for $(typeof(model)).")
-
-"""
-    isfitted(model::StatisticalModel)
-
-Indicate whether the model has been fitted.
-"""
-isfitted(model::StatisticalModel) = error("isfitted is not defined for $(typeof(model)).")
-
-"""
-Fit a statistical model.
-"""
-fit(model::StatisticalModel, args...) = error("fit is not defined for $(typeof(model)).")
-
-"""
-Fit a statistical model in-place.
-"""
-fit!(model::StatisticalModel, args...) = error("fit! is not defined for $(typeof(model)).")
-
-"""
-    aic(model::StatisticalModel)
-
-Akaike's Information Criterion, defined as ``-2 \\log L + 2k``, with ``L`` the likelihood
-of the model, and `k` its number of consumed degrees of freedom
-(as returned by [`dof`](@ref)).
-"""
-aic(model::StatisticalModel) = -2loglikelihood(model) + 2dof(model)
-
-"""
-    aicc(model::StatisticalModel)
-
-Corrected Akaike's Information Criterion for small sample sizes (Hurvich and Tsai 1989),
-defined as ``-2 \\log L + 2k + 2k(k-1)/(n-k-1)``, with ``L`` the likelihood of the model,
-``k`` its number of consumed degrees of freedom (as returned by [`dof`](@ref)),
-and ``n`` the number of observations (as returned by [`nobs`](@ref)).
-"""
-function aicc(model::StatisticalModel)
-    k = dof(model)
-    n = nobs(model)
-    -2loglikelihood(model) + 2k + 2k*(k+1)/(n-k-1)
-end
-
-"""
-    bic(model::StatisticalModel)
-
-Bayesian Information Criterion, defined as ``-2 \\log L + k \\log n``, with ``L``
-the likelihood of the model,  ``k`` its number of consumed degrees of freedom
-(as returned by [`dof`](@ref)), and ``n`` the number of observations
-(as returned by [`nobs`](@ref)).
-"""
-bic(model::StatisticalModel) = -2loglikelihood(model) + dof(model)*log(nobs(model))
-
 """
     r2(model::StatisticalModel)
     r²(model::StatisticalModel)
```

Comment:
Actually I hadn't realized that we haven't moved these to StatsAPI with the rest of functions. I admit these are not mere stubs, but implementations are trivial and lightweight enough that I think it's OK to define them in StatsAPI. Then the situation would be clearer. See https://github.com/JuliaStats/StatsAPI.jl/pull/7.

Reply:
Ok, I'll remove them from this PR as well when your StatsAPI PR is moved and bumped.

Reply:
I've tagged the new release with that change.

Reply:
This is updated to match now.